core/src/main/scala/org/http4s/Method.scala

```diff
@@ -147,10 +147,30 @@ object Method {
 
   private val allByKey: Map[String, Right[Nothing, Method]] = all.map(m => (m.name, Right(m))).toMap
 
-  implicit val catsHashForHttp4sMethod: Hash[Method] = Hash.fromUniversalHashCode
+  implicit val catsInstancesForHttp4sMethod: Show[Method] with Hash[Method] with Order[Method] =
+    new Show[Method] with Hash[Method] with Order[Method] {
+      override def show(t: Method): String =
+        t.toString
 
-  @deprecated("Upgraded to hash. Kept for binary compatibility", "0.21.5")
-  def http4sEqForMethod: Eq[Method] = catsHashForHttp4sMethod
+      override def hash(x: Method): Int =
+        x.hashCode
 
-  implicit val http4sShowForMethod: Show[Method] = Show.fromToString
+      override def compare(x: Method, y: Method): Int =
+        x.name.compare(y.name)
+    }
+
+  @deprecated(
+    message = "Please use catsInstancesForHttp4sMethod. Kept for binary compatibility",
+    since = "0.21.14")
+  def catsHashForHttp4sMethod: Hash[Method] = catsInstancesForHttp4sMethod
```

Comment:
@rossabaker should we drop these since (I'm assuming) this is the 1.x.x branch?

Reply:
Yes.  I like to keep deprecated things around a long time to ease people's migrations, but nobody should be calling these by name.  It's a good chance to clean up.